docset/winserver2025-ps/smbshare/Set-SmbServerConfiguration.md

```diff
@@ -17,21 +17,26 @@ Sets the Server Message Block (SMB) server configuration.
 
 ```
 Set-SmbServerConfiguration [-AnnounceComment <String>] [-AnnounceServer <Boolean>]
- [-AsynchronousCredits <UInt32>] [-AuditSmb1Access <Boolean>] [-AutoDisconnectTimeout <UInt32>]
+ [-AsynchronousCredits <UInt32>] [-AuditClientCertificateAccess <Boolean>]
```

Comment:
These are not in the changes below? It seems like reset and set settings are messed up/divergent here.